packages/util/src/helpers.ts

```diff
@@ -1,5 +1,21 @@
 import { isHexString } from 'ethjs-util'
 
+/**
+ * Checks if a number string is a hex string, if not, converts and returns it 
+ * @param {string} input string to check, convert, and return
```

Comment:
typedoc can infer the type
```suggestion
 * @param input string to check, convert, and return
```